pipelines/broad/reprocessing/cram_to_unmapped_bams/CramToUnmappedBams.wdl

```diff
@@ -20,12 +20,13 @@ workflow CramToUnmappedBams {
     String base_file_name
     String unmapped_bam_suffix = ".unmapped.bam"
     Int additional_disk = 20
+    Bool restore_hardclips = false
```

Comment:
let's leave the default here `true` and I'll be sure to set it to false in the configuration. that way we are not changing the default behaviour of this workflow, just adding an option.